lib/nexus/tests/nexus2ecl.cpp

```diff
@@ -44,6 +44,12 @@ void test_create_ecl_sum(char *root_folder) {
     test_assert_true( util_file_exists( "ECL_CASE.FSMSPEC"));
 
     test_work_area_free(work_area);
+
+
+    ecl_sum = nex::ecl_summary( "/tmp/necl/ECL_CASE", plt );
```

Comment:
This test was quite basic:
1. Please use the `test_area_context` instead of writing to a hardcoded path.
2. Should try to load - and at least do a very basic - something like:
```C
{
   ecl_sum_type * ecl_sum_loaded = ecl_sum_fread_alloc_case("ECL_CASE", ":");

   test_assert_true( ecl_sum_has_key(ecl_sum_loaded, "FOPR"));
   test_assert_time_t_equal( ecl_sum_get_start_time( ecl_sum_loaded ), util_make_data_utc( day, month ,year));
   test_assert_int_equal( ecl_sum_get_data_length( ecl_sum_loaded ), timesteps.size());
   
   for (size_t tstep = 0; tstep < ecl_sum_get_data_ength( ecl_sum_loaded ); tstep++)
        test_assert_double_equal( ecl_sum_get_general_var( ecl_sum_loaded, tstep , "FOPR"), fopr_values[tstep].value);

   ecl_sum_free( ecl_sum_loaded);
}
```

Reply:
Removed the temporary test that wrote a filed to /tmp/necl/ and added the test you wrote for point 2.